pipeline.py

```diff
@@ -72,31 +94,89 @@ class TextProcessingPipeline:
     """
 
     def __init__(self, corpus_manager: CorpusManager):
-        pass
+        self.corpus_manager = corpus_manager
 
     def run(self):
         """
         Runs pipeline process scenario
         """
-        pass
+        articles = self.corpus_manager.get_articles().values()
+        cleaned_tokens = []
+        single_tagged = []
+        multiple_tagged = []
+        for article in articles:
+            raw_text = article.get_raw_text()
+            tokens = self._process(raw_text)
+            for token in tokens:
+                cleaned_tokens.append(token.get_cleaned())
+                single_tagged.append(token.get_single_tagged())
+                multiple_tagged.append(token.get_multiple_tagged())
+            article.save_as(' '.join(cleaned_tokens), 'cleaned')
+            article.save_as(' '.join(single_tagged), 'single_tagged')
+            article.save_as(' '.join(multiple_tagged), 'multiple_tagged')
 
     def _process(self, raw_text: str):
         """
         Processes each token and creates MorphToken class instance
         """
-        pass
+        cleaned_text = ' '.join(re.findall(r'[а-яёА-ЯЁ]+', raw_text))
+        analyzed_cleaned_text = Mystem().analyze(cleaned_text)
+        tokens = []
+        morph = pymorphy2.MorphAnalyzer()
+        for token in analyzed_cleaned_text:
+            if ('analysis' not in token) \
+                    or (not token['analysis']) \
+                    or ('lex' not in token['analysis'][0]
+                        or 'gr' not in token['analysis'][0]):
+                continue
+            morphological_token = MorphologicalToken(token['text'])
+            morphological_token.normalized_form = token['analysis'][0]['lex']
+            morphological_token.tags_mystem = token['analysis'][0]['gr']
+            morphological_token.tags_pymorphy = morph.parse(token['text'])[0].tag
```

Comment:
you have made so many checks for mystem analysis. what about pymorphy - do you trust it more? should be also checked otherwise it is risk prone